src/components/pagination/pagination.js

```diff
@@ -21,24 +21,24 @@ class Pagination extends React.PureComponent {
      * Classname of the container of the pagination, this could be used to customize the inner views
      */
     className: PropTypes.string,
-  }
+  };
```

Comment:
please try to maintain the style of the repo (no semicolon after the class attributes)